tests/property/ic_api/time/src/index.ts

```diff
@@ -0,0 +1,55 @@
+// TODO to improve these tests we would also test init, postUpgrade, and preUpgrade
```

Comment:
how many of these comments apply to time and how many are just copied from trap?

Reply:
Can you access stable memory from inspectMessage? Could we store the time instead of trapping? Trapping is probably fine, I'm mostly curious

Reply:
I will prove that you either can or cannot persist state in inspectMessage

Reply:
Proven